src/devtools/views/SitemapExport.html

```diff
@@ -1,7 +1,10 @@
 <form class="form-horizontal" role="form">
 	<div class="form-group">
 		<div class="col-lg-offset-1 col-lg-10">
-			<textarea rows="7" class="form-control">{{sitemapJSON}}</textarea>
+			<textarea disabled rows="7" class="form-control" id="copyText">
+{{sitemapJSON}}</textarea
+			>
+			<p class="btn btn-primary copyText" data-i18n="sitemap_export_copy_text_button"></p>
```

Comment:
Add button to form-group below instead, give it an id.
Try to place both buttons in one row, or align widths if you stack them vertically.